File: java/AreaOfTriangle.java
import java.util.Scanner;
class AreaOfTriangle
{
	public static void main(String args[])
	{
		Scanner kb=new Scanner(System.in);
		double base,height,area;
		System.out.print("Enter base : ");
		base=kb.nextDouble();

		System.out.print("Enter height : ");
		height=kb.nextDouble();

		area=0.5*base*height;

		System.out.println("Area of triangle is "+area);
		
	}
}

File: c/area-of-triangle.c
#include<stdio.h>
int main()
{
	int base,height;
	float area;
	printf("Enter base : ");
	scanf("%d",&base);
	printf("Enter height : ");
	scanf("%d",&height);
	
	area=0.5*base*height;
	
	printf("\nArea of triangle is %.2f",area);
	return 0;
}
